website/inertia/pages/tickets.tsx

```diff
@@ -0,0 +1,39 @@
+'use client'
+
+import Ticket from '#models/ticket'
+import db from '@adonisjs/lucid/services/db'
+import { Card, CardDescription, CardHeader, CardTitle } from '~/components/ui/card'
+
+const ticketTypes: Ticket[] = await db.from('tickets')
+
+export default function SelectTicketsPage() {
+  const imageSrc = `favicon.svg`
+
+  return (
+    <div className="container mx-auto p-4 flex flex-col items-center">
+      <h1 className="text-3xl font-bold text-center mb-6">Seleciona o teu bilhete</h1>
+      <p className="text-center text-gray-600 mb-8">
+        Seleciona o teu bilhete e clica em comprar para continuar.
+      </p>
+
+      <div className="grid gap-6 grid-cols-1 md:w-1/2">
+        {ticketTypes.map((ticket) => (
+          <a href={`/tickets/${ticket.id}/checkout`}>
```

Comment:
We can use something like [this](https://docs.adonisjs.com/guides/basics/routing#route-identifier) here. But since this is running in the front-end, you need to use the appropriate solution for inertia: https://inertiajs.com/routing (see "Generating URLs" section). Or we can also install Tuyau, which provides this facility in the front-end (https://adonisjs.com/blog/introducing-tuyau#typesafe-links-for-inertia-apps).